src/main/java/com/diffmin/Main.java

```diff
@@ -0,0 +1,42 @@
+package com.diffmin;
+
+import com.diffmin.util.Pair;
+import gumtree.spoon.diff.Diff;
+import java.io.File;
+import java.io.FileNotFoundException;
+import spoon.reflect.CtModel;
+
+/** Main execution of generating and applying patch */
+class Main {
```

Comment:
Codecov says this line has _not_ been covered by tests. What does that mean? We have used it as `Main.main`. Did it want us to instantiate first?

Reply:
Just as a pointer, this is not Codecov's doing, but Jacoco. You can always check the Jacoco report after running tests in `target/site/jacoco/index.html`.

Anyway, I don't know how to interpret that. I'm gonna guess it has something to do with the calls to `System.exit` messing with the instrumentation. Clearly, we're using the class, so it must have been loaded, so just ignore that. There's probably a workaround for this but it's not worth our time :)

Reply:
> There's probably a workaround for this but it's not worth our time :)

Agreed. Let's understand the interpretation later.